### PR TITLE
Gradle & Maven - run tests with cli

### PR DESCRIPTION
**Pull request by Or-Gabay**

### CODE REVIEW

File: utils/tests/utils.go [hunk 1]
```diff
@@ -237,6 +237,25 @@ func (cli *JfrogCli) Exec(args ...string) error {
...
+		output = append(output, v)
+	}
+	if cli.credentials != "" {
+		args := strings.Split(cli.credentials, spaceSplit)
+		os.Args = append(os.Args, args...)
+	}
+
+	log.Info("[Command]", strings.Join(output, " "))
+	return cli.main()
+}
```
Comment (yahavi) [inline utils/tests/utils.go]: This code is almost identical to the above function `Exec`. Can you consolidate them?

File: utils/tests/utils.go [hunk 2]
```diff
@@ -237,6 +237,25 @@ func (cli *JfrogCli) Exec(args ...string) error {
 	log.Info("[Command]", strings.Join(output, " "))
 	return cli.main()
 }
```
Comment (yahavi) [inline utils/tests/utils.go]:
```suggestion
 }

```

File: main_test.go [hunk 3]
```diff
@@ -42,6 +45,7 @@ func setupIntegrationTests() {
 	if *tests.TestNpm || *tests.TestGradle || *tests.TestMaven || *tests.TestGo || *tests.TestNuget || *tests.TestPip {
 		if artifactoryCli == nil {
 			initArtifactoryCli()
+			buildtoolsCreds = authenticate()
```
[the remark this answers is not in the document]
Comment (Or-Gabay) [reply main_test.go]: Removed.

File: utils/tests/utils.go (hunk 1 above)
Comment (Or-Gabay) [reply utils/tests/utils.go]: Consolidate is a bit overkill since we will delete this functionality when cli 2.0 will be released.

**Pull request merged**